src/Mover.ts

```diff
@@ -895,13 +895,64 @@ export class MoverAPI implements Types.MoverAPI {
                 ignoreUncontrolled: true,
                 useActiveModalizer: true,
             });
+            if (isGrid) {
+                // finds the last focusable element on the horizontal
+                next = undefined;
+                let cur: HTMLElement | undefined | null = focusable.findPrev({
```

Comment:
I suppose we can use `findElement()` with `acceptCondition` and avoid the loop — check the `acceptElement` for `visibilityAware` condition above.

Reply:
@mshoho I've updated to use findElement, can you take a look?